etc/kayobe/ansible/wazuh-secrets.yml

```diff
@@ -14,6 +14,54 @@
         path: "{{ wazuh_secrets_path | dirname }}"
         state: directory
 
+    - name: Generate a random password which meets the Wazuh password requirements
```

Comment:
Normally, you would put code like this in a filter plugin. It's good that you dug out the requirements.

Reply:
I think I have a neater way of doing this.